jsonschema/tests/test_validators.py

```diff
@@ -771,6 +772,28 @@ class TestRefResolver(unittest.TestCase):
     stored_uri = "foo://stored"
     stored_schema = {"stored" : "schema"}
 
+    def prev_resolve_fragment(self, document, fragment):
+        fragment = fragment.lstrip(u"/")
```

Comment:
Ok sorry about that, I can remove some of the unnecessary parts of that method. I only put them in there to replicate the original method as closely as possible.